plugins/docsettingtweak.koplugin/_meta.lua

```diff
@@ -0,0 +1,6 @@
+local _ = require("gettext")
+return {
+    name = 'docsettingtweak',
+    fullname = _("Document setting tweak"),
```

Comment:
That sounds a bit weird. Some suggestions:

* Document settings tweaker
* Document settings tweak tool
* Tweak document settings